zulipterminal/config/ui_mappings.py

```diff
@@ -40,3 +40,10 @@
     400: {"bool": "", "name": "Member"},
     600: {"bool": "is_guest", "name": "Guest"},
 }
+
+STREAM_POST_POLICY = {
+    1: "Any user can post.",
+    2: "Only organization administrators can send to this stream.",
+    3: "Guests cannot send to this stream.",
+    4: "Only organization administrators and moderators can send to this stream.",
```

Comment:
Let's skip the trailing `.` - no other fields in the popup have this, and we can always add it uniformly later.